packages/google_maps_flutter/google_maps_flutter_ios/example/ios12/ios/RunnerTests/GoogleMapPolylineController_Test.h

```diff
@@ -0,0 +1,29 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "GoogleMapPolylineController.h"
+
+/**
+ *  Internal APIs exposed for unit testing
+ */
```

Comment:
We've recently standardized on `///` for declaration comments in Obj-C code, so these should all be updated to use that.

(Applies throughout the PR.)